README.md

```diff
@@ -1,28 +1,38 @@
 # Play Next.js Application
 
-PRX audio embed player and landing pages.
+PRX embed audio player and landing pages.
 
 ## Getting Started
 
 Make sure you have **node**, **NPM**, and **yarn** installed and/or updated.
 
-- `node` - 12.22.0 or later - Node.js® is a JavaScript runtime built on Chrome's V8 JavaScript engine.
+- `node` - Node.js® is a JavaScript runtime built on Chrome's V8 JavaScript engine.
 - `yarn` - Wrapper CLI for **npm** that streamlines package retrieval and management.
 
-### Install Node and NPM using NVM
+### Install Node and NPM using `asdf`
 
-It is recommended to install and update using _Node Version Manager_ (**nvm**). Follow [Installation and Update](https://github.com/nvm-sh/nvm/blob/master/README.md#installation-and-update) instructions to get started.
+It is recommended to install and update using _asdf_ (**nvm**). Follow [Installation and Update](https://github.com/PRX/internal/wiki/Guide:-Local-Development-Environment#install-asdf) instructions to get started.
 
-Once **nvm** has been installed, use it to install the latest LTS release of **node** and **npm** by running the following command in your terminal:
+Once asdf has been installed and added to your path, install the Node.js plugin:
 
 ```bash
-nvm install --lts
+asdf plugin add nodejs
+```
+
+Then install the latest stable version of Node and NPM:
+
+```bash
+asdf install nodejs lts
```

Comment:
I don't follow why to do an `install` of a specific Node version, and then do an `asdf install` later on of the version in `.nvmrc`. Seems like it could result in the same issue where Yarn gets installed for one Node environment, and then `yarn dev` gets called in a different one.

I guess my expectation would be that…

```bash
asdf install
npm install --global yarn
yarn
yarn dev
```

…all have to happen together as part of **Setup** to get the correct Node version, with Yarn and all the packages. I could be misremembering how it works, though.

Reply:
I would also considering updating the nvmrc file to match the Node version being used in the Dockerfile, so that local dev always matches the deployed environment

Reply:
Reworked instructions to only use one `asdf install` and removed `.nvmrc` file.